<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openFileWindow</class>
 <widget class="QDialog" name="openFileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface for GraphQL Query</string>
  </property>
  <widget class="QLabel" name="openFileText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse for CSV data file and click Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadFileButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="pathToFileTextbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
